agent-inject/handler_test.go

```diff
@@ -601,13 +401,16 @@ func TestHandlerHandle(t *testing.T) {
 				return
 			}
 
-			var actual []jsonpatch.JsonPatchOperation
+			var actual jsonpatch.Patch
 			if len(resp.Patch) > 0 {
 				req.NoError(json.Unmarshal(resp.Patch, &actual))
 				for i := range actual {
-					actual[i].Value = nil
+					delete(actual[i], "value")
 				}
 			}
+			for i := range tt.Patches {
+				delete(tt.Patches[i], "value")
+			}
```

Comment:
Was this extra `delete` loop for the expected patches necessary with the new jsonpatch, or just being prudent?

Reply:
It's necessary now to remove the value field from the test cases. I tried passing in `nil`, but it still generates some bytes (JSON `null`) I think, so the two won't be equal below.